skein/model.py

```diff
@@ -1152,6 +1152,60 @@ def from_protobuf(cls, obj):
                    security=security)
 
 
+class CredentialProviderSpec(Specification):
```

Comment:
I'd just call this `CredentialProvider`.

Reply:
yeap